rpc/wrtc_signaling_answerer.go

```diff
@@ -151,6 +151,20 @@ func isNetworkError(err error) bool {
 	return true
 }
 
+// logic for computing actual deadline from server-provided deadline.
+func getDeadline(ctx context.Context, logger utils.ZapCompatibleLogger, initStage *webrtcpb.AnswerRequest_Init) (context.Context, func()) {
+	if deadline := initStage.Init.GetDeadline(); deadline != nil {
```

Comment:
I appreciate trying to respect the init stage deadline (or more candidly known as -- some random ass number that has no bearing on reality). I'm inclined to just do a `context.WithTimeout(ctx, 10*time.Second)` here and call it a day (SDKs[1] use 10 seconds).

Feel free to merge this as-is and I can loop back. Appreciate all you've done and don't want this ask to be a burden!

[1] Python until recently was only 5 seconds.

Reply:
I have a slight concern that 5s is not enough https://github.com/viamrobotics/goutils/blob/a21ac14344db965df01d577ad52f690a5213538d/rpc/wrtc_client.go#L157 for the go sdk will always wait at least 5s. I would agree that maybe the easier thing is to log a warning and just use 10s all the time

Reply:
yes, going to merge as is so we can get this into stable RDK

I think you're right from a broader design perspective, but want to give us breathing room to think those thoughts